src/sources/http_server.rs

```diff
@@ -129,6 +130,13 @@ pub struct SimpleHttpConfig {
     #[serde(default = "default_http_method")]
     method: HttpMethod,
 
+    /// Specifies the HTTP response status code on request success.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏

```suggestion
    /// Specifies the HTTP response status code that will be returned on successful requests.
```